### PR TITLE
Update Android Studio

### LINKED ISSUE #471: Support Android Studio 3.3
## Kind (Required)

Android Studio 3.3 is now released on the stable track, so we should also support build tools version 3.3 🎉 

 ## Overview (Required)

- update `Dep.kt#GradlePlugin#android` to version `3.3.0` 
- check for any build failures or unexpected crashes

 ## Links	

- https://developer.android.com/studio/releases/

### PR DESCRIPTION
**Pull request by takahirom**
## Issue
- close #471

## Overview (Required)
- Update Android Studio
- I can not build with Gradle 4.10.3, So I updated to 5.1.1. We should use Android Studio 3.4 for it.
- This version contains this problem (long sync time is needed 😭  ) https://youtrack.jetbrains.com/issue/KT-28128

### CODE REVIEW

**Comment by panpanini** [comment]
it looks related to this... https://issuetracker.google.com/issues/121304903

I tried with `com.android.tools.build:gradle:3.4.0-alpha09` and `com.android.tools.build:gradle:3.4.0-beta01`, but neither worked for me 😭

**Comment by Pluu** [comment]
A new version of "Navigation 1.0.0-alpha10" was released.
https://twitter.com/ianhlake/status/1088179411399004160

File: buildSrc/src/main/java/dependencies/Dep.kt [hunk 1]
```diff
@@ -3,10 +3,10 @@ package dependencies
 @Suppress("unused")
 object Dep {
     object GradlePlugin {
-        val android = "com.android.tools.build:gradle:3.2.1"
+        val android = "com.android.tools.build:gradle:3.3.0"
```
Comment (jmatsu) [inline buildSrc/src/main/java/dependencies/Dep.kt]: > Android Studio 3.4 Beta1 and higher.

but agp is...🤔

Comment (takahirom) [reply buildSrc/src/main/java/dependencies/Dep.kt]: I will check it. thanks 🙏

**Comment by jmatsu** [comment]
> New Number of dex file(s) | 28
> Number of dex file(s) Change | 24

!?!?!?!?!?

**Comment by takahirom** [comment]
Broken 💀 
image

**Comment by takahirom** [comment]
🤔 
Could not resolve com.soywiz:klock:1.1.0.

File: feature/session/src/main/res/layout/fragment_tabular_form_session_pages.xml [hunk 2]
```diff
@@ -13,17 +13,34 @@
...
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
             app:tabGravity="fill"
-            app:tabIndicatorHeight="2dp"
             app:tabIndicatorColor="@color/colorPrimary"
+            app:tabIndicatorHeight="2dp"
             app:tabSelectedTextColor="@color/colorPrimary"
             app:tabTextAppearance="@style/TextAppearance.App.Body1"
+<<<<<<< HEAD
```
Comment (Pluu) [inline feature/session/src/main/res/layout/fragment_tabular_form_session_pages.xml]: Garbage is included.

File: feature/session/src/main/res/layout/fragment_tabular_form_session_pages.xml [hunk 3]
```diff
@@ -13,17 +13,34 @@
...
+        <androidx.viewpager.widget.ViewPager
+            android:id="@+id/tabular_form_sessions_viewpager"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            app:layout_behavior="@string/appbar_scrolling_view_behavior"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/tabular_form_sessions_tab_layout"
+=======
```
Comment (Pluu) [inline feature/session/src/main/res/layout/fragment_tabular_form_session_pages.xml]: Garbage is included.

File: feature/session/src/main/res/layout/fragment_tabular_form_session_pages.xml [hunk 4]
```diff
@@ -34,13 +51,17 @@
             android:layout_height="0dp"
             app:layout_constraintTop_toBottomOf="@+id/tabular_form_sessions_tab_layout"
             app:layout_constraintBottom_toBottomOf="parent"
+>>>>>>> master
```
Comment (Pluu) [inline feature/session/src/main/res/layout/fragment_tabular_form_session_pages.xml]: Garbage is included.

Comment (takahirom) [reply feature/session/src/main/res/layout/fragment_tabular_form_session_pages.xml]: Thanks!

**Comment by takahirom** [comment]
Let me merge 👍

**Comment by takahirom** [comment]
🚀

**Pull request merged**